modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -392,11 +392,26 @@ definition {
 	}
 
 	@description = "LPS-139388 - Verify that is not possible to save a process with the node ID not filled"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CanNotSaveProcessWithEmptyNodeID {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#CanNotSaveProcessWithEmptyNodeID";
+
+		Workflow.openWorkflowListView();
+
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Definition");
+
+		ProcessBuilderKaleoDesignerReact.dragSidebarNodeAndMove(
+			nodeLabel = "Task",
+			nodePosition = "-20,-100",
+			nodeType = "task");
+
+		ProcessBuilderKaleoDesignerReact.editNodeInformationField(
+			infoOption = "Id",
+			infoValue = "");
 
-		// TODO LRQA-72987 CanNotSaveProcessWithEmptyNodeID pending implementation
+		ProcessBuilderKaleoDesignerReact.viewRequiredField(
+			requiredField = "nodeId",
+			save = "true");
 
```

Comment:
Step 5 is missing. Assert the message "Some fields need to be filled.Please fill out the fields before saving or publishing."

Reply:
The Assert this included in the macro `viewRequiredField`

Reply:
:+1: